ibis/backends/dask/execution/__init__.py

```diff
@@ -1,3 +1,12 @@
-# This is awaiting implementation of the rest of execution
-# https://github.com/ibis-project/ibis/issues/2537
-from ibis.backends.pandas.core import execute, execute_node  # noqa: F401,F403
+from .arrays import *  # noqa: F401,F403
+from .decimal import *  # noqa: F401,F403
+from .generic import *  # noqa: F401,F403
+from .join import *  # noqa: F401,F403
+from .maps import *  # noqa: F401,F403
+from .selection import *  # noqa: F401,F403
+from .strings import *  # noqa: F401,F403
+from .structs import *  # noqa: F401,F403
+from .temporal import *  # noqa: F401,F403
+from .timecontext import *  # noqa: F401,F403
+
+# from .window import *  # noqa: F401,F403
```

Comment:
remove this?

Reply:
Yeah - you're right, not planning on getting windows in this PR. Will clean up.